config/config.go

```diff
@@ -578,6 +615,26 @@ func GetString(key string, defaultValue string) (value string) {
 	return viper.GetString(key)
 }
 
+// GetStringSlice is wrapper for viper's GetStringSlice
+func GetStringSlice(key string, defaultValue []string) (value []string) {
+	fmt.Println("TransformKey(key)", TransformKey(key))
```

Comment:
Do we need this log here ?

Reply:
will remove this. kept this for debugging